src/ls_instructions/ls_instructions_from_gates.cpp

```diff
@@ -8,8 +8,9 @@ LSIinstructionFromGatesGenerator::LSIinstructionFromGatesGenerator(IdGenerator&
 : id_generator_(id_generator)
 {}
 
-std::queue<LSInstruction> LSIinstructionFromGatesGenerator::make_t_gate_instructions(PatchId target_id)
+std::queue<LSInstruction> LSIinstructionFromGatesGenerator::make_t_gate_instructions(PatchId target_id, bool is_dagger)
 {
+    // Currently we ignore the is_dagger argument, since the procedure in this case is the same
```

Comment:
Yeah that's a good point. When we have logic for the corrective terms, then it will matter